packages/eslint-plugin/rules/valid-sprintf.js

```diff
@@ -47,10 +71,16 @@ module.exports = {
 						break;
 
 					case 'CallExpression':
+						const argFunctionName =
+							args[ 0 ].callee.property &&
+							args[ 0 ].callee.property.name
+								? args[ 0 ].callee.property.name
+								: args[ 0 ].callee.name;
```

Comment:
Isn't the logic here intended to be served by the `getTranslateFunctionName` utility?

Reply:
Ah yes, I think so 👍